modules/sftmx/components/stats/SftmxStatsVaultsCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { PaginatedTable } from '~/components/table/PaginatedTable';
+import SftmxStatsVaultsHeader from './SftmxStatsVaultsHeader';
+import SftmxStatsVaultsRow from './SftmxStatsVaultsRow';
+import { sortBy } from 'lodash';
+import { useSftmxGetStakingData } from '../../lib/useSftmxGetStakingData';
+import Card from '~/components/card/Card';
+import { VStack, Box, Spinner } from '@chakra-ui/react';
+import { useState, useEffect } from 'react';
+import { GqlSftmxStakingVault } from '~/apollo/generated/graphql-codegen-generated';
+
+export function SftmxStatsVaultsCard() {
+    const { data, loading: isLoading } = useSftmxGetStakingData();
+    const vaults = sortBy(data?.sftmxGetStakingData.vaults, 'unlockTimestamp');
+
+    const skip = 10;
```

Comment:
same as above, move skip to constant outside func, or we define a global default const skip in a utils file somewhere